stories/core/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,132 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  ColorPicker,
+  ColorPickerProps,
+  IconButton,
+  Tooltip,
+} from '../../src/core';
+import { action } from '@storybook/addon-actions';
+import ColorSwatch from '../../src/core/ColorPicker/ColorSwatch';
+import { DefaultColors } from '../../src/core/ColorPicker/ColorPicker';
```

Comment:
Now we have a mix of different kinds of imports:
- module import from barrel (ColorPicker, etc)
- default import from file (ColorSwitch)
- module import from file (DefaultColors)

Here we should think about how and what we expect the user to access. If the user needs something, it should be in the barrel. Also the default import is hiding that bug where the file name does not match the component name.

Reply:
I added ColorSwatch to index.ts so now you can import from src/core like ColorPicker. 

I'm not sure what to do about Default Colors.

Reply:
Well, you could just export DefaultColors too 😄

Reply:
Done